app/assets/javascripts/reports.js

```diff
@@ -267,6 +285,15 @@ function formatRateTooltipText(tooltipItem, data, sumData) {
   return `${percent}% ${label} (${total} patients) in ${date}`;
 }
 
+function formatSubtotalTooltipText(tooltipItem, data, sumData) {
+  const datasetIndex = tooltipItem.datasetIndex;
+  const total = formatNumberWithCommas(Math.ceil(sumData[datasetIndex][tooltipItem.label]));
+  const date = tooltipItem.label;
+  const label = data.datasets[datasetIndex].label.toLowerCase();
+  const percent = Math.ceil(tooltipItem.value);
```

Comment:
Are you sure we always want to round up here?  Why not round? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/round